/study/domain-1-organizational-complexity page fails to load with error

When navigating to /study/domain-1-organizational-complexity, the following error appears:

```
Error: Cannot find module './vendor-chunks/esprima@4.0.1.js'
Require stack:
- /Users/austinbrace/Developer/sa-pro-study-companion/.next/server/webpack-runtime.js
- /Users/austinbrace/Developer/sa-pro-study-companion/.next/server/app/study/[domain]/page.js
- /Users/austinbrace/Developer/sa-pro-study-companion/node_modules/.pnpm/next@14.2.35_react-dom@18.3.1_react@18.3.1__react@18.3.1/node_modules/next/dist/server/require.js
- /Users/austinbrace/Developer/sa-pro-study-companion/node_modules/.pnpm/next@14.2.35_react-dom@18.3.1_react@18.3.1__react@18.3.1/node_modules/next/dist/server/load-components.js
- /Users/austinbrace/Developer/sa-pro-study-companion/node_modules/.pnpm/next@14.2.35_react-dom@18.3.1_react@18.3.1__react@18.3.1/node_modules/next/dist/build/utils.js
- /Users/austinbrace/Developer/sa-pro-study-companion/node_modules/.pnpm/next@14.2.35_react-dom@18.3.1_react@18.3.1__react@18.3.1/node_modules/next/dist/server/dev/static-paths-worker.js
- /Users/austinbrace/Developer/sa-pro-study-companion/node_modules/.pnpm/next@14.2.35_react-dom@18.3.1_react@18.3.1__react@18.3.1/node_modules/next/dist/compiled/jest-worker/processChild.js
```

--- a/next.config.js
+++ b/next.config.js
@@ -7,6 +7,7 @@ const nextConfig = {
       bodySizeLimit: '2mb',
     },
   },
+  serverComponentsExternalPackages: ['js-yaml'],
 }
 
 module.exports = nextConfig

